docs/data/view.md

```diff
@@ -39,6 +39,8 @@ For example, if you specify `Rover Rental` for location and click **SEARCH**, yo
 You can also filter data using the [Viam Python SDK](https://python.viam.dev/).
 For example, you could use the [`BinaryDataByFilter`](/build/program/apis/data-client/#binarydatabyfilter) or [`TabularDataByFilter`](/build/program/apis/data-client/#tabulardatabyfilter) methods to filter binary data or tabular data respectively.
 
+Additionally, you can use the [`viam data dataset` commands](/fleet/cli/#data) to add or remove an image or group of images matching a specific filter using the Viam CLI.
```

Comment:
This is missing context - what are we adding or removing images from?
```suggestion
If you want to create a dataset, you can use the [`viam data dataset` commands](/fleet/cli/#data) to add or remove an image or group of images matching a specific filter using the Viam CLI.
```

Reply:
Hmm, this one's specifically about `filter`-ing the `add` or `remove`, not creating datasets per se. Agreed that it's not clear though -- just pushed a rephrase, what'cha think?